scss/_list-group.scss

```diff
@@ -197,8 +197,20 @@
 //
 // Add modifier classes to change text and background color on individual items.
 // Organizationally, this must come after the `:hover` states.
-@each $color, $value in $background-colors {
-  @include list-group-item-variant($color, $value, color-contrast($value));
+@each $state, $value in $background-colors { // Boosted mod: instead of `@each $state in map-keys($theme-colors)`
+  .list-group-item-#{$state} {
+    --#{$prefix}list-group-color: #{color-contrast($value)}; // Boosted mod: instead of `var(--#{$prefix}#{state}-text)`
+    --#{$prefix}list-group-bg: #{$value}; // Boosted mod: instead of `var(--#{$prefix}#{$state}-bg-subtle)`
+    // Boosted mod: no `--#{$prefix}list-group-border-color`
+
+    &.list-group-item-action {
+      &:hover { // Boosted mod: no `&:focus`
+        --#{$prefix}list-group-action-hover-color: #{color-contrast($value)}; // Boosted mod: instead of `var(--#{$prefix}emphasis-color)`;
+        --#{$prefix}list-group-action-hover-bg: #{if(color-contrast($value) == $white, shade-color($value, 10%), tint-color($value, 10%))}; // Boosted mod: instead of `var(--#{$prefix}#{state}-border-subtle)`;
+      }
+    }
+  }
+  // @include list-group-item-variant($color, $value, color-contrast($value));
```

Comment:
Remove ?

Reply:
Removed in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1757/commits/fcd93a946830e0741557b7e8515bf4998a6fb329